baseapi/apis/rest.py

```diff
@@ -15,11 +15,11 @@
 
 logger = logging.getLogger(__name__)
 DEFAULT_TIMEOUT = 5
```

Comment:
```suggestion
DEFAULT_TIMEOUT = 15
```

I wonder if we should increase the default timeout a bit. I fear the default one could be a bit aggressive for some of our third-party endpoints to respond in time.

Reply:
Changed to 15 although it seems a little excessive imho.

Reply:
Actually, if this is just the connection timeout then it's probably fine as `5`. I thought it might've been the request timeout instead.